app/react/containers/message_form.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { Component, PropTypes } from 'react';
+import axios from 'axios';
+import { connect } from 'react-redux';
+
+class MessageForm extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = { message: '' };
+    this.onFormSubmit = this.onFormSubmit.bind(this);
+    this.onInputChange = this.onInputChange.bind(this);
+  };
+
+  onFormSubmit(event) {
+    event.preventDefault();
+    axios.post('http://localhost:3000/messages', {
+      content: this.state.message,
+      room_id: this.props.activeRoomId,
+    });
```

Comment:
@pewniak747 I wasn't sure how to approach this. In normal conditions when i submit form i'd return created message and append it to the message list. However here i don't really need to update state, as ActionCable is responsible for appending this message (that is being broadcasted). 

Is it ok to add action that doesn't update state @pewniak747?